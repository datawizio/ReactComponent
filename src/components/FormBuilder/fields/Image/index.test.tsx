import "jsdom-global/register";
import React from "react";
import { shallow } from "enzyme";

import { FieldImage } from "./index";

const mockProps = {
  name: "Preview name",
  placeholder: "placeholder",
  label: "label",
  onChange: jest.fn()
};

const setUp = (props?) => shallow(<FieldImage {...props} />);

describe("FormBuilder FieldImage component", () => {
  let component;
  beforeEach(() => {
    component = setUp(mockProps);
  });

  it("Field Image rendered correctly", () => {
    expect(component).toMatchSnapshot();
  });
});
