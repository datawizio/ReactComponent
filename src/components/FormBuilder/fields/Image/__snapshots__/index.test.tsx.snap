// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormBuilder FieldImage component Field Image rendered correctly 1`] = `
<FormItem
  label="label"
  name="Preview name"
>
  <Image
    name="Preview name"
    onChange={[MockFunction]}
    placeholder="placeholder"
  />
</FormItem>
`;
